ozaria/site/components/teacher-dashboard/BaseCurriculumGuide/components/ModuleRow.vue

```diff
@@ -93,6 +97,9 @@ export default {
     getContentTypeHeader () {
       if (this.nameType || this.iconType) {
         const type = this.nameType ? this.nameType : this.iconType
+        if (['ai-use', 'ai-learn'].includes(type)) {
```

Comment:
can we have this array stored in utils somewhere? It's values are used twice in this file itself